### ISSUE TITLE
d1fa845dd36 : can't opam pin + install

### ISSUE THREAD

**Issue by gdsfh** [ygrek/sqlgg#25]
I did `opam pin add sqlgg https://github.com/ygrek/sqlgg.git` (master points to d1fa845dd36), then `opam reinstall sqlgg`, and it failed:
```
# /home/gds/.opam/4.02.1/bin/ocamlopt.opt unix.cmxa -I /home/gds/.opam/4.02.1/lib/ocaml/ocamlbuild /home/gds/.opam/4.02.1/lib/ocaml/ocamlbuild/ocamlbuildlib.cmxa myocamlbuild_config.ml myocamlbuild.ml /home/gds/.opam/4.02.1/lib/ocaml/ocamlbuild/ocamlbuild.cmx -o myocamlbuild
# + /home/gds/.opam/4.02.1/bin/ocamlopt.opt unix.cmxa -I /home/gds/.opam/4.02.1/lib/ocaml/ocamlbuild /home/gds/.opam/4.02.1/lib/ocaml/ocamlbuild/ocamlbuildlib.cmxa myocamlbuild_config.ml myocamlbuild.ml /home/gds/.opam/4.02.1/lib/ocaml/ocamlbuild/ocamlbuild.cmx -o myocamlbuild
# File "myocamlbuild_config.ml", line 1, characters 5-12:
# Error: Unbound module Mybuild
# Command exited with code 2.
```
When I tried to clone and build manually (the same revision), I got another error:
```
ocamlfind ocamlopt -package unix -package ocamlbuild -linkpkg -package mybuild myocamlbuild_config.ml myocamlbuild.ml /home/gds/.opam/4.02.1/lib/ocaml/ocamlbuild/ocamlbuild.cmx -o myocamlbuild
+ ocamlfind ocamlopt -package unix -package ocamlbuild -linkpkg -package mybuild myocamlbuild_config.ml myocamlbuild.ml /home/gds/.opam/4.02.1/lib/ocaml/ocamlbuild/ocamlbuild.cmx -o myocamlbuild
File "myocamlbuild_config.ml", line 5, characters 2-14:
Error: This function has type string -> unit
       It is applied to too many arguments; maybe you forgot a `;'.
Command exited with code 2.
```
How can I install fresh sqlgg?

**Comment by ygrek** [comment]
Second problem is solved by `mybuild` from git.
First one is a bug that ocamlbuild flags are setup only in Makefile, not in oasis (which is used by opam installation).
Thanks for the report!

**Comment by ygrek** [comment]
fixed 9944c4b
